### ISSUE TITLE
Feature suggestion (add 30min option to 30 day display)

### ISSUE THREAD

**Issue by nc9** [opennem/opennem#219]
### Discussed in https://github.com/opennem/opennem/discussions/218

<div type='discussions-op-text'>

<sup>Originally posted by **cy754** March  8, 2023</sup>
I've been trying to find a way of suggesting a feature for the OpenNEM web site, hopefully this is the right place (apologies if not).

It would be great if there was a "30min" option for the "30day"displays. This would equate to 1440 data points, which should be doable considering the 7day/5min combination represents 2016 data points. 

Thanks.</div>

**Comment by nc9** [comment]
thoughts @simonhac

**Comment by simonhac** [comment]
out of interest @nc9, why have we duplicated #218?

**Comment by simonhac** [comment]
ah, got it. it was filed as a suggestion, and you turned it into an issue.

**Comment by nc9** [comment]
yep! there is a new discussions feature I enabled - I didn't know it would use the same ID space though!

https://github.com/features/discussions

**Comment by simonhac** [comment]
makes sense… allows you to reference with the same tagging markdown.
